### DISCUSSION TITLE
Save/Download failed with QFieldCloud

### DISCUSSION THREAD

**Discussion by FuzzyZockt** [opengisch/QField#3498] [Q&A]
If I want to download my qgis project from QFieldCloud to my mobile, I always get the error:
`Downloading error. Packaging job finished unsuccessfully for "name". [QF/no_qgis_project] The project does not contain a valid QGIS project file`
I also get a warning on my pc that reads:
`Warning! Failed to process QGIS project file (.qgs/.qgz). Please check the [job logs]("link") and upload the project file again.`

Now to the part where I get confused. In the QFieldCloud repository is everything I need. I can just download it to my pc (before deleted the project form my local machine) and it works fine. I can download it and move it to my mobile and it works fine too. But I can't download my files directly through the app using the QFieldCloud feature.
I tried making a new project and updating my .qgs file, but it didn't help.

**Comment by arilena** [comment]
I have the same issue. It started recently, and until now I have had no problem.

**Reply by suricactus** [reply]
Please follow this thread https://github.com/opengisch/QField/discussions/3753
